### PR TITLE
Support search defines for name resolution of program objects.

### PR DESCRIPTION
**Pull request by rsbeckerca**
Fixes: #94

Signed-off-by: Randall S. Becker <randall.becker@nexbridge.ca>

### CODE REVIEW

**Comment by rsbeckerca** [comment]
This is not yet working. Do not approve yet

File: MANUAL.md [hunk 1]
```diff
@@ -470,7 +485,7 @@ This License applies to any manual or other work, in any medium, that contains a
...
+A "Secondary Section" is a named appendix or a front-matter section of the Document that deals exclusively with the relationship of the publishers or authors of the Document to the Document��������s overall subject (or to related matters) and contains nothing that could fall directly within that overall subject. (Thus, if the Document is in part a textbook of mathematics, a Secondary Section may not explain any mathematics.) The relationship could be a matter of historical connection with the subject or with related matters, or of legal, commercial, philosophical, ethical or political position regarding them.
```
[the remark this answers is not in the document]
Comment (rsbeckerca) [reply MANUAL.md]: Should be better now.

File: MANUAL.md [hunk 2]
```diff
@@ -450,7 +465,7 @@ The following extensions are currently defined for GMake:
 Appendix A GNU Free Documentation License
 Version 1.3, 3 November 2008
 
-Copyright © 2000, 2001, 2002, 2007, 2008 Free Software Foundation, Inc.
+Copyright �� 2000, 2001, 2002, 2007, 2008 Free Software Foundation, Inc.
```
[the remark this answers is not in the document]
Comment (rsbeckerca) [reply MANUAL.md]: Should be fixed

**Comment by rsbeckerca** [comment]
Looks good now. `--search-define`  is working and resolving TAL from a proper search define. Fails with error 11 if unresolved.

**Review by Jojo-Schmitz — APPROVED** [review]

**Pull request merged**